crates/index-scheduler/src/error.rs

```diff
@@ -109,6 +109,8 @@ pub enum Error {
     InvalidIndexUid { index_uid: String },
     #[error("Task `{0}` not found.")]
     TaskNotFound(TaskId),
+    #[error("Task `{0}` does not provide any content file.")]
```

Comment:
```suggestion
    #[error("Task `{0}` does not contain any documents. Only `documentAdditionOrUpdate` tasks with the statuses `enqueued` or `processing` contain documents")]
```

Hey, what do you think of this more specific message to reduce questions when someone use it wrong?